src/services/inplayer.notifications.service.ts

```diff
@@ -0,0 +1,59 @@
+import InPlayer from '@inplayer-org/inplayer.js';
+
+import { reloadActiveSubscription, logout } from '#src/stores/AccountController';
+
+interface Notification {
+  type: NotificationsTypes;
+}
+
+export enum NotificationsTypes {
+  ACCESS_GRANTED = 'access.granted',
+  ACCESS_REVOKED = 'access.revoked',
+  SUBSCRIBE_SUCCESS = 'subscribe.success',
+  SUBSCRIBE_FAILED = 'subscribe.failed',
+  PAYMENT_CARD_SUCCESS = 'payment.card.success',
+  PAYMENT_CARD_FAILED = 'payment.card.failed',
+  PAYMENT_CARD_REQUIRES_ACTION = 'payment.card.requires.action',
+  ACCOUNT_LOGOUT = 'account.logout',
+}
+
+const handleLogout = async () => {
+  await logout();
```

Comment:
Do we expect these methods to grow in the future?  If not, why not just call `logout` and `reloadActiveSubscription` directly?

Reply:
If they are called directly it throws an error saying something like `trying to call before initialise`.... however this file will be updated, more of them are like placeholders just so we have the idea how it will work.